main.yml

```diff
@@ -228,6 +228,21 @@ rhel:
       - id: remediations
   top_level: true
 
+ruledev:
+  title: Rule Development
+  channel: '#flip-mode-squad'
+  deployment_repo: https://github.com/RedHatInsights/ruledev-frontend-build
+  frontend:
+    title: Rule Development
+    paths:
+      - /ruledev
+    sub_apps:
+      - id: 'ruledev'
```

Comment:
If you add a sub app with an ID that points to the original app, we'll end up with an infinite loop when trying to build the navigation.